tests/unit/test_publish_decorator.py

```diff
@@ -125,7 +125,6 @@ def my_serializer_1(obj):
         self.assertEqual("", published[1].body["last_name"])
         self.assertEqual("", published[1].body["first_name"])
         self.assertIsNone(published[1].body["last_login"])
-        self.assertAlmostEqual(date_joined.strftime("%Y-%m-%dT%H:%M:%S.%f")[:-3], published[1].body["date_joined"])
```

Comment:
nitpick: what do you think about use a [freezegun](https://pypi.org/project/freezegun/0.3.4/) ever than remove the line ?

Reply:
In my view freezegun is a fantastic lib and it would really be a good idea to use it. But I think that in this case it would not be worth it because it is a single line of information that is not so critical. But like I said, it's my vision. Would you like to add another lib as a dependency for this test?